/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:25:31.662828 47320090125568 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:25:31.662828 47320090125568 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:31.662828 47320090125568 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:31.678177 47320090125568 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 02:25:31.678177 47320090125568 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:31.678177 47320090125568 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:31.686953 47320090125568 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:25:31.686953 47320090125568 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:31.686953 47320090125568 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:31.715150 47320090125568 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:25:31.715150 47320090125568 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:31.715150 47320090125568 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 02:25:38.539795 47320090125568 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 02:25:38.539795 47320090125568 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 02:25:38.539795 47320090125568 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 02:25:38.539795 47320090125568 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 02:25:38.539795 47320090125568 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 02:25:38.539795 47320090125568 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 02:25:41.500190 47320090125568 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:25:41.500190 47320090125568 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:41.500190 47320090125568 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 734
V0421 02:25:41.500190 47320090125568 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:41.528331 47320090125568 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 02:25:41.528331 47320090125568 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:41.528331 47320090125568 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 734
V0421 02:25:41.528331 47320090125568 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:41.540257 47320090125568 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:25:41.540257 47320090125568 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:41.540257 47320090125568 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 734
V0421 02:25:41.540257 47320090125568 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:41.579229 47320090125568 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:25:41.579229 47320090125568 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:41.579229 47320090125568 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 734
V0421 02:25:41.579229 47320090125568 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:41.596656 47320090125568 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 02:25:41.596656 47320090125568 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:41.596656 47320090125568 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 610                                       
V0421 02:25:41.817498 47320090125568 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:25:41.817498 47320090125568 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:41.817498 47320090125568 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 734
V0421 02:25:42.566828 47320090125568 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 02:25:42.566828 47320090125568 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:42.566828 47320090125568 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 734, actual 176
V0421 02:25:42.588022 47320090125568 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 02:25:42.588022 47320090125568 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:42.588022 47320090125568 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 734, actual 176
V0421 02:25:42.616148 47320090125568 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 02:25:42.616148 47320090125568 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:42.616148 47320090125568 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 734, actual 176
V0421 02:25:42.632667 47320090125568 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 02:25:42.632667 47320090125568 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:42.632667 47320090125568 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 734, actual 176
V0421 02:25:42.698351 47320090125568 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 02:25:42.698351 47320090125568 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:42.698351 47320090125568 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 734, actual 176
V0421 02:25:42.944861 47320090125568 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 02:25:42.944861 47320090125568 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:42.944861 47320090125568 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 734, actual 176
V0421 02:25:42.944861 47320090125568 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 610                                       
V0421 02:25:43.188147 47320090125568 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:25:43.188147 47320090125568 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:43.188147 47320090125568 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 734, actual 176
V0421 02:25:43.188147 47320090125568 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 1468
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:25:47.323973 47320090125568 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:25:47.323973 47320090125568 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:47.323973 47320090125568 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:25:47.323973 47320090125568 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:25:47.323973 47320090125568 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:25:47.370181 47320090125568 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 02:25:47.370181 47320090125568 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:47.370181 47320090125568 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:25:47.609996 47320090125568 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:47.609996 47320090125568 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:47.609996 47320090125568 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:47.702390 47320090125568 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:47.702390 47320090125568 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:47.702390 47320090125568 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:47.702390 47320090125568 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:50.726745 47320090125568 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:50.726745 47320090125568 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:50.726745 47320090125568 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 923789.187272523                          
V0421 02:25:50.726745 47320090125568 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:50.726745 47320090125568 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:52.777756 47320090125568 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:52.777756 47320090125568 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.777756 47320090125568 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923792.211484342                          
V0421 02:25:52.777756 47320090125568 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923789.187272523                          
V0421 02:25:52.777756 47320090125568 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:52.777756 47320090125568 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923794.262429247                          
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923792.211484342                          
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923789.187272523                          
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:52.847610 47320090125568 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923794.332450424                          
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923794.262429247                          
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923792.211484342                          
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923789.187272523                          
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:52.853821 47320090125568 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:52.915766 47320090125568 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:25:52.915766 47320090125568 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.915766 47320090125568 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:52.942640 47320090125568 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:25:52.942640 47320090125568 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.942640 47320090125568 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:52.962661 47320090125568 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:25:52.962661 47320090125568 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.962661 47320090125568 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:52.982634 47320090125568 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:25:52.982634 47320090125568 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:52.982634 47320090125568 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.000515 47320090125568 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:25:53.000515 47320090125568 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.000515 47320090125568 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.020474 47320090125568 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:25:53.020474 47320090125568 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.020474 47320090125568 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.070505 47320090125568 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:25:53.070505 47320090125568 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.070505 47320090125568 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.084327 47320090125568 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:25:53.084327 47320090125568 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.084327 47320090125568 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.102464 47320090125568 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:25:53.102464 47320090125568 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.102464 47320090125568 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.118816 47320090125568 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 02:25:53.118816 47320090125568 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.118816 47320090125568 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 500                                        
V0421 02:25:53.395800 47320090125568 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:25:53.395800 47320090125568 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.395800 47320090125568 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:25:53.395800 47320090125568 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.395800 47320090125568 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:25:53.395800 47320090125568 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:25:53.452630 47320090125568 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 02:25:53.452630 47320090125568 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.452630 47320090125568 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:25:53.490223 47320090125568 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 02:25:53.490223 47320090125568 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.490223 47320090125568 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:25:53.500638 47320090125568 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 02:25:53.500638 47320090125568 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.500638 47320090125568 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:25:53.512359 47320090125568 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 02:25:53.512359 47320090125568 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.512359 47320090125568 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:25:53.526260 47320090125568 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 02:25:53.526260 47320090125568 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.526260 47320090125568 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923794.338651906                          
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923794.332450424                          
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923794.262429247                          
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923792.211484342                          
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923789.187272523                          
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:53.572469 47320090125568 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923789.082075                             
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923795.057205319                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923794.338651906                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923794.332450424                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923794.262429247                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923792.211484342                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923789.187272523                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923789.094884977                          
V0421 02:25:53.578778 47320090125568 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923789.082075                             
W0421 02:25:53.578937 47320090125568 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 02:25:53.578937 47320090125568 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 02:25:53.578937 47320090125568 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 923789.082075                             
W0421 02:25:53.578937 47320090125568 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 02:25:53.578937 47320090125568 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:25:54.828047 47320090125568 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 02:25:54.828047 47320090125568 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.828047 47320090125568 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:54.854968 47320090125568 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 02:25:54.854968 47320090125568 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.854968 47320090125568 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 611, actual 616
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:25:54.871106 47320090125568 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 02:25:54.871106 47320090125568 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.871106 47320090125568 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:54.886899 47320090125568 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 02:25:54.886899 47320090125568 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.886899 47320090125568 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:54.928039 47320090125568 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:25:54.928039 47320090125568 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.928039 47320090125568 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:54.928039 47320090125568 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 611                                         
V0421 02:25:54.928039 47320090125568 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 611                                         
V0421 02:25:54.959737 47320090125568 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:25:54.959737 47320090125568 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.959737 47320090125568 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:54.959737 47320090125568 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 611                                       
V0421 02:25:54.959737 47320090125568 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 611                                       
V0421 02:25:54.973548 47320090125568 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:25:54.973548 47320090125568 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:54.973548 47320090125568 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 611                                         
V0421 02:25:58.200558 47320090125568 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 02:25:58.200558 47320090125568 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:58.200558 47320090125568 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 611                                       
V0421 02:25:58.218266 47320090125568 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 02:25:58.218266 47320090125568 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:58.218266 47320090125568 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 611                                       
V0421 02:25:58.218266 47320090125568 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 734, actual 743
V0421 02:25:58.476541 47320090125568 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:25:58.476541 47320090125568 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:58.476541 47320090125568 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:25:58.476541 47320090125568 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 734, actual 743
V0421 02:25:58.476541 47320090125568 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 734, actual 743
V0421 02:25:58.715671 47320090125568 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:25:58.715671 47320090125568 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:58.715671 47320090125568 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 743
V0421 02:25:58.715671 47320090125568 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:58.715671 47320090125568 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 611                                         
V0421 02:25:58.715671 47320090125568 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 743
V0421 02:25:58.755175 47320090125568 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:25:58.755175 47320090125568 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:58.755175 47320090125568 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 743
V0421 02:25:58.755175 47320090125568 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:25:58.755175 47320090125568 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 611                                       
V0421 02:25:58.755175 47320090125568 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 743
V0421 02:25:58.776405 47320090125568 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 02:25:58.776405 47320090125568 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:58.776405 47320090125568 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 1820, actual 1838
V0421 02:25:59.137195 47320090125568 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 02:25:59.137195 47320090125568 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.137195 47320090125568 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 611                                            
V0421 02:25:59.369788 47320090125568 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 02:25:59.369788 47320090125568 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.369788 47320090125568 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:59.388222 47320090125568 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 02:25:59.388222 47320090125568 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.388222 47320090125568 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:59.405888 47320090125568 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 02:25:59.405888 47320090125568 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.405888 47320090125568 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:59.678397 47320090125568 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 02:25:59.678397 47320090125568 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.678397 47320090125568 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:59.702115 47320090125568 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 02:25:59.702115 47320090125568 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.702115 47320090125568 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:59.719450 47320090125568 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 02:25:59.719450 47320090125568 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.719450 47320090125568 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 611, actual 616
V0421 02:25:59.945488 47320090125568 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 02:25:59.945488 47320090125568 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:25:59.945488 47320090125568 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.026007 47320090125568 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:26:00.026007 47320090125568 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.026007 47320090125568 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.026007 47320090125568 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.044485 47320090125568 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:26:00.044485 47320090125568 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.044485 47320090125568 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.044485 47320090125568 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.063438 47320090125568 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:26:00.063438 47320090125568 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.063438 47320090125568 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.063438 47320090125568 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.083835 47320090125568 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:26:00.083835 47320090125568 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.083835 47320090125568 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.083835 47320090125568 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.101072 47320090125568 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:26:00.101072 47320090125568 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.101072 47320090125568 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.101072 47320090125568 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.120480 47320090125568 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:26:00.120480 47320090125568 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.120480 47320090125568 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.120480 47320090125568 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.164635 47320090125568 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:26:00.164635 47320090125568 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.164635 47320090125568 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.164635 47320090125568 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.179403 47320090125568 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:26:00.179403 47320090125568 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.179403 47320090125568 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.179403 47320090125568 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.197472 47320090125568 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:26:00.197472 47320090125568 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.197472 47320090125568 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 02:26:00.197472 47320090125568 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.213977 47320090125568 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 02:26:00.213977 47320090125568 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.213977 47320090125568 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 611                                        
V0421 02:26:00.213977 47320090125568 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 611                                        
V0421 02:26:00.393513 47320090125568 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 02:26:00.393513 47320090125568 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.393513 47320090125568 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.393513 47320090125568 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.439161 47320090125568 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 02:26:00.439161 47320090125568 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.439161 47320090125568 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.439161 47320090125568 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.451373 47320090125568 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 02:26:00.451373 47320090125568 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.451373 47320090125568 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.451373 47320090125568 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.464881 47320090125568 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 02:26:00.464881 47320090125568 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.464881 47320090125568 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.464881 47320090125568 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.484226 47320090125568 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 02:26:00.484226 47320090125568 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.484226 47320090125568 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.579567 47320090125568 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 02:26:00.579567 47320090125568 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.579567 47320090125568 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:00.652304 47320090125568 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 02:26:00.652304 47320090125568 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:00.652304 47320090125568 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:01.404037 47320090125568 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 02:26:01.404037 47320090125568 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:01.404037 47320090125568 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 611                                        
V0421 02:26:02.746046 47320090125568 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 02:26:02.746046 47320090125568 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:02.746046 47320090125568 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.032768 47320090125568 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 02:26:03.032768 47320090125568 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:03.032768 47320090125568 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.815129 47320090125568 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 02:26:03.815129 47320090125568 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:03.815129 47320090125568 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.926418 47320090125568 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:26:03.926418 47320090125568 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:03.926418 47320090125568 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.926418 47320090125568 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.926418 47320090125568 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.948096 47320090125568 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:26:03.948096 47320090125568 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:03.948096 47320090125568 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.948096 47320090125568 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.948096 47320090125568 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.969588 47320090125568 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:26:03.969588 47320090125568 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:03.969588 47320090125568 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.969588 47320090125568 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.969588 47320090125568 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.991318 47320090125568 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:26:03.991318 47320090125568 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:03.991318 47320090125568 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:03.991318 47320090125568 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:03.991318 47320090125568 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.010918 47320090125568 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:26:04.010918 47320090125568 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:04.010918 47320090125568 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.010918 47320090125568 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:04.010918 47320090125568 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.032588 47320090125568 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:26:04.032588 47320090125568 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:04.032588 47320090125568 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.032588 47320090125568 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:04.032588 47320090125568 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.082740 47320090125568 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:26:04.082740 47320090125568 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:04.082740 47320090125568 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.082740 47320090125568 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:04.082740 47320090125568 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.436765 47320090125568 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:26:04.436765 47320090125568 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:04.436765 47320090125568 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.436765 47320090125568 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:04.436765 47320090125568 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.456120 47320090125568 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:26:04.456120 47320090125568 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:04.456120 47320090125568 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:26:04.456120 47320090125568 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 611, actual 616
V0421 02:26:04.456120 47320090125568 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:04.930207 47320090125568 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:04.930207 47320090125568 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:04.930207 47320090125568 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:04.930207 47320090125568 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 611                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:08.643838 47320090125568 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:08.643838 47320090125568 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:08.643838 47320090125568 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 609                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:08.643838 47320090125568 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:08.643838 47320090125568 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 611                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:12.336228 47320090125568 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:12.336228 47320090125568 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:12.336228 47320090125568 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 608                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:12.336228 47320090125568 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 609                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:12.336228 47320090125568 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:12.336228 47320090125568 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 611                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 603                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 608                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 609                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:16.057638 47320090125568 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 611                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 625                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 603                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 608                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 609                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:19.774054 47320090125568 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 611                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 614                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 625                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 603                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 608                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 609                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:23.502402 47320090125568 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 611                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 605                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 614                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 625                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 603                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 608                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 609                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 616                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:26:27.206056 47320090125568 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 611                                         
W0421 02:26:27.206300 47320090125568 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 02:26:27.206300 47320090125568 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 02:26:27.206300 47320090125568 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 611                                         
W0421 02:26:27.206300 47320090125568 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 02:26:27.206300 47320090125568 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/pubmed/SAGE/seed_1/sparsity_before.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight        250000    0.0000
1    _orig_mod.encoder.graph_layer_1.bias           500  100.0000
2  _orig_mod.encoder.graph_layer_2.weight         64000    0.0000
3    _orig_mod.encoder.graph_layer_2.bias           128  100.0000
4      _orig_mod.encoder.decoder_1.weight        250000    0.0004
5        _orig_mod.encoder.decoder_1.bias           500    0.0000
6      _orig_mod.encoder.decoder_2.weight        250000    0.0000
7        _orig_mod.encoder.decoder_2.bias           500    0.0000
8         _orig_mod.encoder.linear.weight           384    0.0000
9           _orig_mod.encoder.linear.bias             3    0.0000
Saved sparsity snapshot to outputs_compiled/tran/pubmed/SAGE/seed_1/sparsity_after.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight        250000  0.250800
1    _orig_mod.encoder.graph_layer_1.bias           500  0.000000
2  _orig_mod.encoder.graph_layer_2.weight         64000  0.021875
3    _orig_mod.encoder.graph_layer_2.bias           128  0.000000
4      _orig_mod.encoder.decoder_1.weight        250000  0.000400
5        _orig_mod.encoder.decoder_1.bias           500  0.000000
6      _orig_mod.encoder.decoder_2.weight        250000  3.039200
7        _orig_mod.encoder.decoder_2.bias           500  2.000000
8         _orig_mod.encoder.linear.weight           384  0.000000
9           _orig_mod.encoder.linear.bias             3  0.000000
Saved forward timing report to outputs_compiled/tran/pubmed/SAGE/seed_1/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.000482      20039
1   _orig_mod.encoder.graph_layer_1             0.000684      20039
2   _orig_mod.encoder.graph_layer_2             0.000618      20039
3       _orig_mod.encoder.decoder_1             0.000554      20039
4       _orig_mod.encoder.decoder_2             0.000522      20039
5          _orig_mod.encoder.linear             0.000283      20039
6   _orig_mod.encoder.vq.project_in             0.000470      20039
7  _orig_mod.encoder.vq.project_out             0.000585      20039
8    _orig_mod.encoder.vq._codebook             0.000625      20039
CPU Memory Usage: 1269.11 MB
Max GPU Memory Used: 1244.60 MB
1244.59 MB
